src/openms_gui/source/VISUAL/APPLICATIONS/TOPPViewBase.cpp

```diff
@@ -732,12 +732,23 @@ namespace OpenMS
         OPENMS_LOG_INFO << "INFO: done loading all " << std::endl;
 
         // a mzML file may contain both, chromatogram and peak data
-        // -> this is handled in PlotCanvas::addPeakLayer
-        data_type = LayerDataBase::DT_CHROMATOGRAM;
-        if (peak_map_sptr->containsScanOfLevel(1))
+        // -> this is handled in PlotCanvas::addPeakLayer FIXME: No it's not!
```

Comment:
@poshul I would love to review this part, but to be honest I am not sure what this change means for now.. Could you give me a brief explanation?

Reply:
Ah, that's a leftover from another PR that was merged after I started this one. I just updated this branch and this should go away.